elcid/utils.py

```diff
@@ -96,7 +96,7 @@ def find_patients_from_mrns(mrns):
     e.g. 000 will be removed.
     """
     cleaned_mrn_to_mrn = {
-        i.strip().lstrip('0'): i for i in mrns if i and i.strip().lstrip('0')
+        i.strip().lstrip('0'): i for i in mrns if i is not None and i.strip().lstrip('0')
```

Comment:
Are you sure this change is required?
(Besides it being close to the upper bound for happy dictionary comprehensions, why did the test on L63 pass before this change?)